src/caret_analyze/record/records_service/response_time.py

```diff
@@ -759,6 +745,42 @@ def _to_timeseries(self, input_column, output_column):
 
         return t_in, latency
 
+    def to_best_case_records(self) -> RecordsInterface:
+        records = self._records.to_range_records()
+        input_column = records.columns[1]
+        output_column = records.columns[-1]
+        return self._to_records(input_column, output_column)
+
+    def to_worst_case_records(self) -> RecordsInterface:
+        records = self._records.to_range_records()
+        input_column = records.columns[0]
+        output_column = records.columns[-1]
+        return self._to_records(input_column, output_column)
+
+    def _to_records(self, input_column, output_column) -> RecordsInterface:
+        records: RecordsInterface = self._create_empty_records(input_column)
+
+        range_records = self._records.to_range_records()
+        t_in = range_records.get_column_series(input_column)
+        t_out = range_records.get_column_series(output_column)
```

Comment:
It's better to change t_in/out to timestamps_in/out or somethings more comprehensible.

Reply:
This PR will not be changed in this PR as it is written in accordance with existing code.